### ISSUE TITLE
missing outputs

### ISSUE THREAD

**Issue by hullsean** [azavea/terraform-aws-vpc#15]
Hi, I'm excited to get this module working.

I followed the readme, created almost the exact module setup.

However, I can't see the outputs you've defined with terraform output.

I tried "terraform refresh" also, but nothing displays.  I was using a shared state file, so I switched to local tfstate (unconfigured backend).  

still the outputs are missing.

Any thoughts on what could cause this?

**Comment by hectcastro** [comment]
Hey @hullsean, are you trying to use them as [module outputs](https://www.terraform.io/docs/modules/usage.html#outputs) or the higher level [output variables](https://www.terraform.io/intro/getting-started/outputs.html)? We use the former a bunch with these modules, but only the latter at a Terraform project level.

**Comment by hullsean** [comment]
So I had a couple of issues, and I've been posting to the terraform google group.

1. Outputs not defined in the module, cannot be *ADDED*.  This just isn't supported with Terraform.

2. In my module, I must create outputs of *YOUR* outputs in order to get access to those variables.

I created a git repo of my own to test all of this.  Then I referenced my git repo with a "source =" line.  I was having a terrible time troubleshooting, as the outputs still didn't show up.

Turns out "terraform init" caches the repository tf files *ONLY ONCE*.  It continues to use these old files even if you do "terraform init" again.  In my case I had subsequently added to the repo and changed the source.  The solution there is:

$ terraform init -get=true -upgrade=true

Terraform is rather unintuitive around the topic of "OUTPUTS".

Thank you for your help.

-Sean
